templates/docs/examples/patterns/side-navigation/_layout-application.html

```diff
@@ -1,6 +1,6 @@
 {# used by the application layout example #}
 
-<div class="p-side-navigation--icons {% if is_dark %}is-dark{% endif %}" id="drawer-icons">
+<nav class="p-side-navigation--icons {% if is_dark %}is-dark{% endif %}" id="drawer-icons">
```

Comment:
It feels a bit inconsistent that we use `<nav>` here on root element of the pattern, while on other cases on `.p-side-navigation__drawer`. Maybe we should do it always on the root element?